packages/realm-web/CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 ?.?.? Release notes (2020-??-??)
 =============================================================
 
+### Breaking Changes
```

Comment:
are we releasing with these on top. Why not add them after the release ?

Reply:
Right. I agree this should probably have been added on a later commit.